pkgs/applications/audio/zrythm/default.nix

```diff
@@ -0,0 +1,175 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, SDL2
+, alsa-lib
+, libaudec
+, bash
+, bash-completion
+, breeze-icons
+, carla
+, chromaprint
+, cmake
+, curl
+, dconf
+, epoxy
+, ffmpeg
+, fftw
+, fftwFloat
+, flex
+, git
+, glib
+, gtk3
+, gtksourceview3
+, guile
+, graphviz
+, help2man
+, json-glib
+, jq
+, libbacktrace
+, libcyaml
+, libgtop
+, libjack2
+, libpulseaudio
+, libsamplerate
+, libsndfile
+, libsoundio
+, libxml2
+, libyaml
+, lilv
+, lv2
+, meson
+, ninja
+, pandoc
+, pcre
+, pcre2
+, pkg-config
+, python3
+, reproc
+, rtaudio
+, rtmidi
+, rubberband
+, serd
+, sord
+, sratom
+, texi2html
+, wrapGAppsHook
+, xdg-utils
+, xxHash
+, vamp-plugin-sdk
+, zstd
+}:
+
+let
+  pname = "zrythm";
+  version = "1.0.0-alpha.26.0.13";
+in
+stdenv.mkDerivation {
+  inherit pname;
+  inherit version;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "zrythm";
  version = "1.0.0-alpha.26.0.13";```

Reply:
Is there a reason that this is preferred? It seems like they're equivalent, and I've seen `rec` referred to as an anti-pattern (i.e. https://nix.dev/anti-patterns/language#rec-expression)
I'm happy to change it though.